shapely/constructive.py

```diff
@@ -129,41 +130,41 @@ def buffer(
 
     Examples
     --------
-    >>> from shapely import LineString, Point, Polygon
+    >>> from shapely import LineString, Point, Polygon, BufferCapStyle, BufferJoinStyle
     >>> buffer(Point(10, 10), 2, quadsegs=1)
     <POLYGON ((12 10, 10 8, 8 10, 10 12, 12 10))>
     >>> buffer(Point(10, 10), 2, quadsegs=2)
     <POLYGON ((12 10, 11.414 8.586, 10 8, 8.586 8.586, 8 10, 8.5...>
     >>> buffer(Point(10, 10), -2, quadsegs=1)
     <POLYGON EMPTY>
     >>> line = LineString([(10, 10), (20, 10)])
-    >>> buffer(line, 2, cap_style="square")
+    >>> buffer(line, 2, cap_style='square')
```

Comment:
It also doesn't matter too much, but so this change is not needed (I suppose it is a left-over from changing this and then changing back)